File: libs/net/entities/DataSourceScheduleType.cs
namespace TNO.Entities;

/// <summary>
/// Provides data source schedule type.
/// </summary>
public enum DataSourceScheduleType
{
    /// <summary>
    /// This data source has no schedule.
    /// </summary>
    None = 0,

    /// <summary>
    /// Repeating will run a continuously repeating schedule.
    /// </summary>
    Continuous = 1,

    /// <summary>
    /// Has a start and stop time each day.
    /// </summary>
    Daily = 2,

    /// <summary>
    /// Advanced schedule has many start and stop times.
    /// </summary>
    Advanced = 3,
}

File: app/editor/src/hooks/api-editor/constants/DataSourceScheduleType.ts
export enum DataSourceScheduleType {
  /** This data source has no schedule. */
  None = 0,
  /** Repeating will run a continuously repeating schedule. */
  Continuous = 1,
  /** Has a start and stop time each day. */
  Daily = 2,
  /** Advanced schedule has many start and stop times. */
  Advanced = 3,
}
